presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -42,8 +42,17 @@
     private double cpuCostWeight = 0.75;
     private double memoryCostWeight = 0;
     private double networkCostWeight = 0.25;
+
+    public static class JoinDistributionType
```

Comment:
How do enum config properties work.  Is there a way to give a good error message if a value is invalid?